src/analog/commands/downlink/GetExAbsArchiveHoursMC.ts

```diff
@@ -94,7 +93,12 @@ class GetExAbsArchiveHoursMC extends Command {
         buffer.setHours(hour, hours);
         buffer.setChannels(channelList.map(index => ({index} as IChannel)));
 
-        return Command.toBytes(COMMAND_ID, buffer.toUint8Array());
+        return Command.toBinary(COMMAND_ID, buffer.toUint8Array());
+    }
+
+    // returns full message - header with body
+    toBytes (): Uint8Array {
+        return this.toBinary().buffer;
```

Comment:
seems toBytes now does nothing but wraping toBinary

do we need it?

Reply:
fixed